textual_search_over_documets/text_search_over_documets.py

```diff
@@ -0,0 +1,45 @@
+from text_search_over_documets_abc_meta import TextSearchDocumentABCMeta
+from elasticsearch import Elasticsearch
+from enum import Enum
+
+
+@TextSearchDocumentABCMeta.register
+class TextSearchDocuments:
```

Comment:
the syntax we usually use is:
`class TextSearchDocuments(TextSearchDocumentABCMeta):`